scripts/plan.py

```diff
@@ -0,0 +1,82 @@
+from github import Github 
+from github import Auth
+import os
+import sys
+import logging
+
+logger = logging.getLogger(__name__)
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)s] %(asctime)s - %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S"
+)
+
+try:
+    pull_request_id = int(os.getenv("CIRCLE_PULL_REQUEST").split("/")[-1])
+    commit_hash = os.getenv("CIRCLE_SHA1")
+    token = os.getenv("GITHUB_TOKEN")
+    auth = Auth.Token(token)
+
+    g = Github(auth=auth)
+
+    org_name = "Cyber4All"
+    org = g.get_organization(org_name)
+
+    repo = g.get_repo("Cyber4All/Security-Injections")
+    pull_request = repo.get_pull(pull_request_id)
+    commits = pull_request.get_commits()
+    logger.info("Authorization complete, Repository and Pull Request retrieved.")
+except:
```

Comment:
Can we log e so we know what the issue is if this occurs.
```suggestion
except e:
```